src/driver/mysql/MysqlDriver.ts

```diff
@@ -217,11 +217,17 @@ export class MysqlDriver implements Driver {
             case ColumnTypes.BOOLEAN:
                 return value === true ? 1 : 0;
             case ColumnTypes.DATE:
-                return moment(value).format("YYYY-MM-DD");
+                if (moment(value).isValid())
+                  return moment(value).format("YYYY-MM-DD");
+                else return '0000-00-00';
```

Comment:
it seems to be a developer preference on using 0000-00-00 vs null. something to consider, that I just tested, is moment(null) returns NaN .. while Date(null) returns the current timestamp. Wouldn't both Null and 0000-00-00 both hydrate as invalid date anyways?